app/preprints/-components/submit/review/template.hbs

```diff
@@ -1,3 +1,67 @@
 <div local-class='preprint-input-container  {{if (is-mobile) 'mobile'}}'>
-    {{t 'preprints.submit.step-five.title'}}
+    <div local-class='step-container'>
+        <h3 local-class='title'
+            data-test-title-step-one
+        >
+            {{t 'preprints.submit.step-one.title'}}
+        </h3>
+
+        <div local-class='content-container'>
+            <div local-class='title'>
+                {{t 'preprints.submit.step-five.select-a-service'}}
```

Comment:
"Select a service" here seems a bit weird here, as this step is to just review the preprint before submission. Could ask Product folks for the label, but something like "Provider" or "Service" may be more appropriate here